draft-sullivan-tls-implicit-ech.md

```diff
@@ -209,6 +280,19 @@ Supporting implicit ECH configurations limits the number of different ECH
 keys supported by a server on the same IP address since the outer SNI and
 config_id can no longer be used to choose the appropriate ECH configuration.
 
+Deployments can support key rotation using the following strategy:
+ * Add a new hashed public key to their ImplicitECHConfig
```

Comment:
It looks like the procedures you define won't allow this to be propagated without causing ECH failures.  At least in the transition from public_name to key hashes.

It wasn't obvious to me that the text on Enterprise policy was intended to encompass the existing public_name-based retry method.  If that is the case, then this probably needs to be clearer.